test/Superpower.Tests/Parsers/SpanTests.cs

```diff
@@ -83,5 +84,33 @@ public void RegexMatches()
             var r = parser(input);
             Assert.Equal("Foo", r.Value.ToStringValue());
         }
+
+        [Theory]
+        [InlineData("", "")]
```

Comment:
Shouldn't `Until("STOP")` fail if `"STOP" is missing from the input string?

Reply:
Yep, I moved it to the fail expecting test set, and removed the OptionalOrDefault()